src/serac/mesh_utils/CMakeLists.txt

```diff
@@ -4,32 +4,34 @@
 #
 # SPDX-License-Identifier: (BSD-3-Clause)
 
-set(mesh_headers
+# Mesh Utilities
+
+set(mesh_utils_headers
     mesh_utils.hpp
     mesh_utils_base.hpp
     )
 
-set(mesh_sources
+set(mesh_utils_sources
     mesh_utils.cpp
     )
 
-set(mesh_depends serac_infrastructure)
+set(mesh_utils_depends serac_infrastructure)
 
 blt_add_library(
-    NAME        serac_mesh
-    HEADERS     ${mesh_headers}
-    SOURCES     ${mesh_sources}
-    DEPENDS_ON  ${mesh_depends}
+    NAME        serac_mesh_utils
```

Comment:
The motivation for renaming `serac_mesh` -> `serac_mesh_utils` is that with the introduction of `serac::Mesh` makes it is hard to differentiate what is "serac mesh". Adding "utils" makes it clearer that the CMake target just includes helper mesh functions and doesn't include `serac::Mesh`.